kernel/configs/fragments/common/virtio.conf

```diff
@@ -22,5 +22,6 @@ CONFIG_SCSI_LOWLEVEL=y
 CONFIG_SCSI_VIRTIO=y
 CONFIG_VIRTIO_BLK=y
 CONFIG_TTY=y
+CONFIG_VIRTIO_BALLOON=y
```

Comment:
I'm not sure if we actually use balloon - @jcvenegas can you check/config pls.

Reply:
Dont think we use this today.

Reply:
okay, I'll delete it. ;)